packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/container.tsx

```diff
@@ -35,16 +40,83 @@ const SignupConfirmCodeContainer = ({
     keyFetchToken,
     unwrapBKey,
   } = location.state || {};
+  const navigate = useNavigate();
+
+  const storedLocalAccount: StoredAccountData | undefined = currentAccount();
+  const { email, sessionToken } = storedLocalAccount || {};
 
   const { finishOAuthFlowHandler, oAuthDataError } = useFinishOAuthFlowHandler(
     authClient,
     integration
   );
 
-  if (!integration) {
-    return (
-      <LoadingSpinner className="bg-grey-20 flex items-center flex-col justify-center h-screen select-none" />
-    );
+  const navigateToContentServer = useCallback(
+    (path: string, hasBounced?: boolean) => {
+      const params = new URLSearchParams(location.search);
+
+      // Passing back the 'email' param causes various behaviors in
+      // content-server since it marks the email as "coming from a RP".
+      // Also remove `emailFromContent` since we pass that when coming
+      // from content-server to Backbone, see Signup container component
+      // for more info.
+      params.delete('emailFromContent');
+      params.delete('email');
```

Comment:
I think we want to keep these `delete`s in case a user goes from the Signup page to this page and then back to `index`, but I don't think we need to pass the params from the Backbone router.